src/main/java/org/folio/rmapi/RMAPIService.java

```diff
@@ -325,6 +327,21 @@ public Long getFirstProviderElement(Vendors vendors) {
       .thenCompose(vendorResult -> CompletableFuture.completedFuture(vendorResult.getVendor()));
   }
 
+  public CompletableFuture<PackageResult> retrievePackage(PackageId packageId, List<String> includedObjects) {
+    CompletableFuture<PackageByIdData> packageFuture = retrievePackage(packageId);
+    CompletableFuture<Titles> titlesFuture;
+    if (includedObjects.contains(INCLUDE_RESOURCES_VALUE)) {
```

Comment:
is it a possibility that this "includedObjects" can be null?

Reply:
no, if include flag is null then "includedObjects" will be set to empty list

Reply:
Sounds good!